src/FriendlyExceptionInterface.php

```diff
@@ -9,6 +9,8 @@
 /**
  * Provides a friendly name and a possible solution for an exception.
  * Error handlers may consider the interface to render additional information right at the error screen.
+ *
+ * @api
```

Comment:
What is it for?

Reply:
https://psalm.dev/docs/annotating_code/supported_annotations/#api-psalm-api